runtime/common/src/xcm.rs

```diff
@@ -272,7 +282,7 @@ pub type LocationToAccountId<RelayNetwork> = (
 	// Straight up local `AccountId32` origins just alias directly to `AccountId`.
 	AccountId32Aliases<RelayNetwork, AccountId>,
 	// Generate remote accounts according to polkadot standards
-	cfg_primitives::xcm::HashedDescriptionDescribeFamilyAllTerminal<AccountId>,
+	HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>,
```

Comment:
Important change here: [slack thread](https://kflabs.slack.com/archives/C05TBFRBL15/p1711112291272849)

Reply:
We had test covering the conversions made beforehand and can ensure that they stay the same, right?

Reply:
The test we had that were copied from [here](https://github.com/moonbeam-foundation/polkadot/blob/d83bb6cc7d7c93ead2fd3cafce0e268fd3f6b9bc/xcm/xcm-builder/src/location_conversion.rs#L749) (the polkadot fork of frontier) no longer compiles with the new type because the way this type is used has changed. But in the fork of the frontier of `polkadot-v1.1.0` this "custom" type no longer exists and given the comment they have on top of the type it seems like it's no longer needed in favor of this new one.

Not sure if it convinces you. I was not involved when this type was added, and I can not tell you more about...